package.json

```diff
@@ -24,23 +24,38 @@
   },
   "homepage": "https://github.com/odoo/jabberwock#readme",
   "devDependencies": {
-    "@types/jest": "^24.0.18",
+    "@types/chai": "^4.2.1",
+    "@types/dom-inputevent": "^1.0.5",
```

Comment:
Interesting. I created an InputEvent interface in our project because it was missing from TypeScript. I guess it can be deleted if we now depend from inputevent ?

Reply:
yep I think so